### ISSUE TITLE
[4.17.1] Lockscreen timer setting out of focus when system XL font size enabled

### ISSUE THREAD

**Issue by bungabunga** [signalapp/Signal-Android#7496]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
I have extra large system font size enabled since I don't see well. If I try to set the lockscreen timer on Signal 4.17.1 the right column of numbers (3, 6, 9, 00) is out of screen. It wasn't on 4.17.0. 

### Steps to reproduce
- enable large font screen in android display settings (or accessibility settings)
- try to set up Lockscreen timer on Signal

**Actual result:** Numbers 3, 6, 9, 00 out of screen
**Expected result:** They should be on the settings screen

### Screenshots
![lockscreen_timer](https://user-images.githubusercontent.com/2994878/37248065-f036cd1c-24c7-11e8-9cb6-56c498922cd2.png)

### Device info
**Device:** Sony Xperia Z3c
**Android version:** 6.0.1
**Signal version:** 4.17.1

**Comment by moxie0** [comment]
I've filed this here: https://github.com/svenwiegand/time-duration-picker/issues/16
